File: Tokopedia/Classses/UIRefactor/Home/MyWishlist/MyWishlistShopReputation.m
//
//  MyWhishlistMojitoShopReputation.m
//  Tokopedia
//
//  Created by Billion Goenawan on 9/23/16.
//  Copyright © 2016 TOKOPEDIA. All rights reserved.
//

#import "MyWishlistShopReputation.h"

@implementation MyWishlistShopReputation

    +(NSDictionary *)attributeMappingDictionary
    {
        NSArray *keys = @[@"score",@"set", @"level", @"image"];
        ;
        return [NSDictionary dictionaryWithObjects:keys forKeys:keys];
    }

    +(RKObjectMapping*)mapping
    {
        RKObjectMapping *mapping = [RKObjectMapping mappingForClass:self];
        [mapping addAttributeMappingsFromDictionary: [self attributeMappingDictionary]];
        return mapping;
    }

@end

File: Tokopedia/Classses/UIRefactor/Home/MyWishlist/MyWishlistShopReputation.swift
//
//  MyWishlistShopReputation.swift
//  Tokopedia
//
//  Created by Billion Goenawan on 10/14/16.
//  Copyright © 2016 TOKOPEDIA. All rights reserved.
//

import UIKit

@objc(MyWishlistShopReputation)
class MyWishlistShopReputation: NSObject {
    
    var score: NSNumber!
    var set: String!
    var level: NSNumber!
    var image: String!
    
    class func mapping() -> RKObjectMapping {
        let mapping = RKObjectMapping(for: MyWishlistShopReputation.self)
        
        mapping?.addAttributeMappings(from:["score", "set", "level", "image"])
        return mapping!
    }
}
